### ISSUE TITLE
[BUG] missing themes after upgrade to 1.0.66

### ISSUE THREAD

**Issue by somera** [aristocratos/bpytop#316]
**Describe the bug**

After upgrade to 1.0.66 the themes folder ist empty.

```
$ ls -al $HOME/.config/bpytop/themes
total 8
drwxrwxr-x 2 rafcio rafcio 4096 Mär 12 22:00 .
drwxrwxr-x 3 rafcio rafcio 4096 Mär 12 22:05 ..
```

**To Reproduce**

[Steps to reproduce the behavior:]

**Expected behavior**

[A clear and concise description of what you expected to happen.]

**Screenshots**

[If applicable, add screenshots to help explain your problem.]

**Info (please complete the following information):**
 - bpytop version: `bpytop -v`
bpytop version: 1.0.66
 - psutil version: `bpytop -v` (version 5.7.0 or above is required)
psutil version: 5.8.0
 - (Linux) Linux distribution and version:
Distributor ID: Ubuntu
Description:    Ubuntu 20.04.2 LTS
Release:        20.04
Codename:       focal
 - (OSX/FreeBSD) Os release version:
 - Terminal used:
 - Font used:
 - Python version, `python3 --version` (version 3.6 or above is required):
Python 3.8.5

**Additional context**

contents of `~/.config/bpytop/error.log`

`02/06/21 (12:24:50) | WARNING: No theme named "gruvbox_dark" found!`

(try running bpytop with `--debug` flag if error.log is empty)

![image](https://user-images.githubusercontent.com/8334250/120486032-f3fe7000-c3b4-11eb-8711-73ac7c39d3bb.png)

**Comment by ajgringo619** [comment]
[Linux Mint 20.1]
I lost my themes as well. However, on my system the themes are now simply ignored; my  `$HOME/.config/bpytop/themes` folder is still populated as normal.

**Comment by aristocratos** [comment]
@somera @ajgringo619 
What install methods are you using?
There is nothing in the latest update that's even related to themes.

Depending on your install method, the themes will be installed in `/usr/local/share/bpytop/themes`, `/usr/share/bpytop/themes` or your python3 distpackages folder if installed with `pip3`. 
The `$HOME/.config/bpytop/themes` directory is only for user installed themes and never written to by bpytop.

**Comment by somera** [comment]
@aristocratos this is my

`pip3 install bpytop --upgrade`

upgrade call.

**Comment by aristocratos** [comment]
@somera 
What's the output when you run `pip3 --version` and `which -a bpytop`.
The first command should give you the path to the `site-packages` directory were `bpytop-themes` gets installed by pip3. Also look carefully that the reported python version from pip3 is the same as the version you get when you run `python3 --version`.
And the `which -a bpytop` command should show if you got multiple versions installed.

**Comment by somera** [comment]
```
$ pip3 --version
pip 21.1.2 from /home/rafcio/.local/lib/python3.8/site-packages/pip (python 3.8)
$ python3 --version
Python 3.8.5
```

**Comment by aristocratos** [comment]
@somera @ajgringo619 
I think I've found the issue. For some unknown reason the themes folder don't have the correct name in the PyPi package, not sure how that happened since the folder structure where it packages from is the same as it always has been.

But a quick fix for now until next release is:
Run `pip3 --version` to see the path to the site-packages folder, go there and rename `themes` folder to `bpytop-themes`.

**Comment by ajgringo619** [comment]
I can do that, but how does that affect the themes I've been using in `$HOME/.config/bpytop/themes`?

**Comment by somera** [comment]
> @somera @ajgringo619
> I think I've found the issue. For some unknown reason the themes folder don't have the correct name in the PyPi package, not sure how that happened since the folder structure where it packages from is the same as it always has been.
> 
> But a quick fix for now until next release is:
> Run `pip3 --version` to see the path to the site-packages folder, go there and rename `themes` folder to `bpytop-themes`.

Thx. The fix works. After that I have to select the right theme again.

**Comment by ajgringo619** [comment]
OK, as with @somera this works for me, but only with the built-in themes. None of my added themes work.

**Comment by aristocratos** [comment]
@ajgringo619 
As long as you didn't rename the `themes` folder in `~/.config/bpytop/` (only the folder in site-packages should be renamed) it should work as normal. No changes have been made in the program itself to mess with the themes, only the folder name in the PyPi package was changed.

So if the above is true and your custom themes are correctly formatted it would be a different bug and you'll need to open a new bug report following the existing template.

**Comment by ajgringo619** [comment]
I think the problem was the format of the themes. I had downloaded a bunch of themes for XFCE (Base16) and just copied a couple over to the `$HOME/.config/bpytop/themes directory`, expecting them to work.

Consider my issue solved.

**Comment by aristocratos** [comment]
@ajgringo619 
Ok. I think most of all existing themes for bpytop are included in the install since anyone creating a new theme are free to add it to the repo.
I haven't seen any themes "in the wild" myself, but if you do and they are formatted for either bashtop or bpytop, they should work.

**Comment by fritogotlayed** [comment]
I don't know if it's that I have rebooted my computer since updating `bpytop` or something else but I do not have themes in my home directory config nor do I have themes in the site-packages area. Are there any plans to cut a hot fix style release to get the themes back out there or shoudl I just want for vNext?

**Comment by aristocratos** [comment]
Fixed in v1.0.67

**Comment by elliottregan** [comment]
My themes directory is still empty. I'm running bpytop v1.0.67 on macOS 11.4, and installed through Homebrew.
